srcpkgs/gcc/template

```diff
@@ -353,6 +353,10 @@ do_install() {
 	ln -sfr ${DESTDIR}/usr/include/c++/${_minorver} \
 		${DESTDIR}/usr/include/c++/${_patchver}
 
+	# Install libbacktrace. (it's not installed by makefiles)
+	vinstall libbacktrace/.libs/libbacktrace.a \
+		644 usr/lib/gcc/${_triplet}/${_minorver}
+
```

Comment:
This should be its own commit.

I am not enthused about installing something that comes from a `.libs` directory into the package. Are there even headers for this?

Reply:
Well, not even sure if this is the route I want to take. I'd much prefer upstream libbacktrace, but upstream doesn't do versioning (no tags, no releases, just a master branch on a git repo).

Yep, I know, but the gcc source doesn't install libbacktrace at all. This is inspired by how Debian handle libbacktrace. 

Yes, headers are available...sorry, my bad. Will include them.

Any thoughts regarding my request for feedback?